addon-resizer/Makefile

```diff
@@ -53,7 +68,7 @@ buildx-setup:
 container: .container-$(ARCH)
 .container-$(ARCH): buildx-setup
 	cp -r * $(TEMP_DIR)
-	cd $(TEMP_DIR) && sed -i 's|BASEIMAGE|$(BASEIMAGE)|g' Dockerfile
+	cd $(TEMP_DIR) && sed -i- 's|BASEIMAGE|$(BASEIMAGE)|g' Dockerfile
```

Comment:
What does this do?

Reply:
`make container` wasn’t running without this syntax fix. It is no longer required here as this has already been tested out. I will revert this change.